include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -235,6 +235,8 @@ WARNING(cannot_assign_value_to_conditional_compilation_flag,none,
 ERROR(error_optimization_remark_pattern, none, "%0 in '%1'",
       (StringRef, StringRef))
 
+ERROR(error_some_error_occured_in_a_file_that_was_used_by_this_one,none, "some error occured in a file that was used by this one", ())
```

Comment:
Sorry, this still isn't the correct description. All we know is that an error occurred somewhere—maybe by another file in this batch, maybe in a file used by this file, maybe in a file used by another file in the batch. That is, because dia files are per-input in batch mode, we need this error even if *no* diagnostics are suppressed.

Reply:
Fixed (I think).